apps/condo/domains/banking/components/BankAccountReport.tsx

```diff
@@ -0,0 +1,401 @@
+import { BankAccountReport as BankAccountReportType } from '@app/condo/schema'
+import styled from '@emotion/styled'
+import { Row, Col } from 'antd'
+import dayjs from 'dayjs'
+import ReactECharts from 'echarts-for-react'
+import find from 'lodash/find'
+import get from 'lodash/get'
+import isEmpty from 'lodash/isEmpty'
+import React, { useState, useCallback, useRef, useMemo, useEffect } from 'react'
+
+import { useIntl } from '@open-condo/next/intl'
+import { Tabs, Card, Typography, Select, Option, Space } from '@open-condo/ui'
+import type { TypographyTextProps } from '@open-condo/ui'
+import type { CardProps } from '@open-condo/ui'
+
+import { BankAccountReport as BankAccountReportClient } from '@condo/domains/banking/utils/clientSchema'
+import { BasicEmptyListView } from '@condo/domains/common/components/EmptyListView'
+import { TotalBalanceIcon, BalanceOutIcon, BalanceInIcon } from '@condo/domains/common/components/icons/TotalBalance'
+import { useLayoutContext } from '@condo/domains/common/components/LayoutContext'
+import { Loader } from '@condo/domains/common/components/Loader'
+
+import type { BankAccount as BankAccountType } from '@app/condo/schema'
+import type { RowProps } from 'antd'
+import type { EChartsOption, EChartsReactProps } from 'echarts-for-react'
+
+const BANK_ACCOUNT_REPORT_ROW_GUTTER: RowProps['gutter'] = [40, 40]
+const LABEL_TRUNCATE_LENGTH = 17
+const CHART_COLOR_SET = ['#E45241', '#9036AA', '#4154AE', '#4595EC', '#51B7D1', '#419488', '#96C15B', '#D1DB58', '#FBE960', '#F5C244', '#F29B38']
```

Comment:
What are these colors? Why not from our UI-kit?

Reply:
Maybe it could be a colors from ui-kit. I'll check this out

Reply:
So I double checked this question & these colors didn't match any string from UI-kit